overlord/snapstate/snapmgr.go

```diff
@@ -560,6 +604,12 @@ func (m *SnapManager) Ensure() error {
 
 	m.runner.Ensure()
 
+	// needs to run after the runner
```

Comment:
I agree with @mvo5 that this shouldn't be needed due to how apparmor is setup during boot. @zyga, are we missing something?